fieldservice_sale/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "depends": [
         "fieldservice",
         "sale_management",
-        "fieldservice_account",
```

Comment:
Why this change ?

Reply:
We have used customer_id which is introduced in fieldservice_account_analytic.

Reply:
Please remove the introduction of customer_id.